notebooks/lesson05_random-forest-deep-dive.ipynb

```diff
@@ -0,0 +1,1577 @@
+{
```

Comment:
<p>maybe it's worth noting at this point that Bagging is a two step process:</p><ol><li>Bootstrapping</li><li>Aggregation</li></ol><p>--&gt; <strong>B</strong>ootstrap <strong>AGG</strong>regat<strong>ING</strong></p><p>And so the last step would be to combine the outputs of each forest.</p><p>(Thx Chris Albon: <a href="https://images.squarespace-cdn.com/content/v1/55dbfccee4b08731143170b6/1539363521188-2JR5C19SA6MH1DF8O0T8/ke17ZwdGBToddI8pDm48kCnSkN-eXZSR5YvvB_R36KhZw-zPPgdn4jUwVcJE1ZvWQUxwkmyExglNqGp0IvTJZUJFbgE-7XRK3dMEBRBhUpyvTRJqce8tcz4gYQhIzwn4wxbYWGTPJQiVcKj2YF2g9RpcCrSKkdUlrE-CdXP11Ig/Bagging_web.png" target="_blank">link</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
done!